### PR TITLE
Added class selectedDay (near hasEvents) in calendar if day has been …

### PR DESCRIPTION
**Pull request by c2po**
Hi Tim,
in the calendar view I wanted to highlight the day which the user had selected.
Therefore the class selectedDay is added to the day (near hasEvents) if selected.
Also I had to change the format string to get german month names.

### CODE REVIEW

File: Classes/Controller/CalendarController.php [hunk 1]
```diff
@@ -190,15 +190,18 @@ public function yearAction($year = null)
...
      * @param int $year
      * @param int $month
+     * @param int $day
      *
      * @return void
      */
-    public function monthAction($year = null, $month = null)
+    public function monthAction($year = null, $month = null, $day = null)
     {
         $date = DateTimeUtility::normalizeDateTime(1, $month, $year);
```
Comment (lochmueller) [inline Classes/Controller/CalendarController.php]: Please use the "$date" for that handling and do not create a new $today variable:
$date = DateTimeUtility::normalizeDateTime($day, $month, $year);

**Comment by lochmueller** [comment]
Nice patch. Thanks... please take a look at the note. We do not need a new variable for that handling. You can just add the $day argument and pass it to the $date logic.

**Comment by lochmueller** [comment]
I will fix it directly in the master

**Comment by lochmueller** [comment]
FYI: I streamline the code: https://github.com/lochmueller/calendarize/commit/e3316a793e4b3a30ff526777bce017baaabe828f
Futhermore I add the selectedDay class also for days, that have no events.
In the AbstractLoop i fixed the bug, that the loop view Helper manipualte the date variable of the template.

Thanks you for the patch

**Comment by c2po** [comment]
Thank you :)

**Pull request merged**